xcode/Subconscious/Shared/Models/Identified.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Identified.swift
+//  Subconscious (iOS)
+//
+//  Created by Gordon Brander on 3/2/23.
+//
+
+import Foundation
+
+/// A wrapping type that is identifiable.
+/// Use to wrap types that can't conform to identifiable.
+struct Identified<Value>: Identifiable, Equatable
```

Comment:
Allows us to add a persistent identity to records that might otherwise not have one.